reference/forms/types/password.rst

```diff
@@ -69,6 +69,15 @@ object.
             'mapped' => false,
         ]);
 
+``toggle``
+~~~~~~~~~~
+**type**: ``boolean`` **requires**: `symfony/ux-toggle-password`_
+
+Adds "Show"/"Hide" links to the field which toggle the password field to plaintext when clicked.
+See `symfony/ux-toggle-password`_ for more details.
+
+.. _`symfony/ux-toggle-password`: https://symfony.com/bundles/ux-toggle-password/current/index.html
```

Comment:
One last thing: I think we usually put all these link lines at the bottom of the file

Reply:
Yes we do.

Maybe an idea for a new DOCtor rule @alamirault ?

Reply:
I agree. I created issue https://github.com/OskarStark/doctor-rst/issues/1512